redfish-core/lib/ethernet.hpp

```diff
@@ -1974,6 +2037,14 @@ inline void requestEthernetInterfacesRoutes(App& app)
 
             if (dhcpv4 || dhcpv6)
             {
+                // TODO this is workaround needed to avoid ipv4 static and DHCP
+                // address co-existence This behaviour should be fixed in
+                // backend networkd D-bus application
+                if (dhcpv4)
+                {
+                    // Delete IPv4 static addesses while enabling DHCPv4
+                    deleteIPv4StaticAddresses(ifaceId, asyncResp);
+                }
```

Comment:
This can be moved into the handleDHCPPatch method before invoking setDHCPEnabled

Reply:
done